packages/bpk-component-slider/src/BpkSlider.module.scss

```diff
@@ -18,20 +18,24 @@
 
 @import '~bpk-mixins';
 
-$slider-control-inset-size: 20 * $bpk-one-pixel-rem;
+$bpk-spacing-v2: true;
+
+$slider-control-inset-size: 22 * $bpk-one-pixel-rem;
+$slider-large-control-inset-size: bpk-spacing-sm() * 7;
 
 .bpk-slider {
-  height: $bpk-spacing-lg;
+  height: bpk-spacing-xl();
   -webkit-tap-highlight-color: transparent;
 
   &--large {
-    height: $bpk-spacing-xxl;
+    height: bpk-spacing-xxl();
```

Comment:
should one of these be width or what effect does defining height twice have?

Reply:
Ah this was one value I missed whilst testing sizes out and forgot to clean up! Good spot, it just needs to be one height definition